traffic_ops/testing/api/v4/servers_test.go

```diff
@@ -885,6 +886,7 @@ func UpdateTestServers(t *testing.T) {
 
 	originalHostname := *resp.Response[0].HostName
 	originalXMPIDD := *resp.Response[0].XMPPID
+	originalMTU := *resp.Response[0].Interfaces[0].MTU
```

Comment:
This check should go down below the check for `len(infs)` on line 896 to avoid a segfault. For the same reason, should also check that MTU is not `nil` before dereferencing it

Reply:
ok